### DISCUSSION TITLE
함수 모양 통일

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#98] [General]
- [ ] const 화살표 함수
- [ ] function
- [ ] export를 어디에 쓸 것인가

**Comment by JuhyeokC** [comment]
컴포넌트 형태는 `function` 으로

기능을 하는 함수는 `const` 로
```
export default function Conponent() {

  const handleAction = () => {

  }

  return (
    
  )
}
```

**Comment by JiinSeok** [comment]
1. 컴포넌트 export default function ()
2. hook export default function ()
3. 기능함수 export const ()

import Component from 'Component'
import customHook from 'customHook'
import { func } from 'util' //유틸함수
